### DISCUSSION TITLE
Patch Update for the Switch platform by THQ

### DISCUSSION THREAD

**Discussion by Quintus24** [AmProsius/gothic-1-community-patch#385] [General]
Hi guys,

Just posting this here in case it might be useful for the project in some regards.
THQ seems to have fixed a lot of the bugs, on the Switch platform, that are already fixed or reported by your team with G1CP.

Thou I don't know yet if they plan to release it for the PC/Steam version as well.
Anyway, the bug list is interesting and fun to read nonetheless.

https://thqnordic.com/article/gothic-1-switch-list-fixes
https://twitter.com/gothicthegame/status/1687085880291270656

Warm Regards,
Quintus24

**Comment by szapp** [comment]
Thank you for bringing that to our attention (I was not aware of this)! I am very curious how they implemented these changes and whether they recompiled the game and reparsed scripts - or used some sort of post-hoc injections like the G1CP is doing. The exact implementation may indicate whether there are plans to also release a new version of the PC-version.

The list of fixes may prove to be quite valuable. In case that post may get lost or the link becoming unreachable I copied the current contents at time of me writing this at the end of this post.

After skimming through the list some lines caught my attention.

----

> - The Templars at Orc Cemetary are now deceased by default.

Unfortunate. I believe our approach is much more immersive (#115).

> - Stone will now be hostile in Chapter 5.

I hope they made *Skip* hostile.

> - The key to the storage room at the castle smithy will now appear in Skip's inventory, as it should.

"As it should" is quite an assumption.

> - The Rice Lord can now be traded with.
> - Bartholo can now be traded with.
> - Gorn can now be traded with after the convoy raid.

It seems to me that the trade dialogs had been purposely disabled in all cases. For Gorn's dialog it even gives a reason, why trading with him should not be allowed. I guess who ever orchestrated these "fixes", does not (care to) understand the German comment
https://github.com/AmProsius/gothic-1-community-patch/blob/45b0a07c9472d432999e39392cab9dc3ba99a2cd/scriptbase/_work/Data/Scripts/Content/Story/Missions/DIA_PC_Fighter.d#L245

<details><summary>Code</summary>
<p>

https://github.com/AmProsius/gothic-1-community-patch/blob/45b0a07c9472d432999e39392cab9dc3ba99a2cd/scriptbase/_work/Data/Scripts/Content/Story/Missions/DIA_BAU_900_Ricelord.d#L84-L107

https://github.com/AmProsius/gothic-1-community-patch/blob/45b0a07c9472d432999e39392cab9dc3ba99a2cd/scriptbase/_work/Data/Scripts/Content/Story/Missions/DIA_EBR_106_Bartholo.d#L57-L81

https://github.com/AmProsius/gothic-1-community-patch/blob/45b0a07c9472d432999e39392cab9dc3ba99a2cd/scriptbase/_work/Data/Scripts/Content/Story/Missions/DIA_PC_Fighter.d#L230-L253

</p>
</details> 

> - Fixed problems with the unused blackmailers.
> - The unused blackmailers now spawn at the bridge near the New Camp.
> - The unused tramps now spawn. 
> - The unused guard guarding the gate of the Old Mine now spawns. 

I think these don't qualify as fixes, much like the ones above. This rather seems like deliberately cut content.

> - There are no longer more people than there are beds. 

What is the fix here? More beds or less people?

> - Friendly NPCs will now defend themselves when attacked.

I can see all sorts of issues arising from that.

> - Baal Lukor can now be found in the Sect Camp prior to Chapter 3. 
> - Gor Boba Lukor can now be found in the Sect Camp prior to Chapter 3. 
> - The gate to the exchange place now closes in Chapter 4.

Interesting...

> - Re-added canyon near the Old Mine.

I wonder what canyon that is and what does re-added mean. Sounds again like a case of restoring what was deliberately cut content.

> - Cavalorn will now become angry if you loot his chest.

He always struct me as a rather calm and passive guy. It doesn't suit his character in my opinion.

---


<details><summary>List of all fixes</summary>
<p>
Source: https://thqnordic.com/article/gothic-1-switch-list-fixes

Gothic Classic Fixes

=== AI fixes ===
- Attitudes will now persist upon saving the game and reloading. 
- When killed by a script, NPCs will now lose all their weapons, as intended. 
- NPCs will now always correctly pick up their dropped weapon after a fight. 
- The fleeing voiceline will now change depending on whether the NPC has been defeated before or not. 
- Fixed an oversight that caused NPCs to generate new ammo even if they still had some in their inventory. 
- Fixed inconsistent behavior of Orcs when having the UluMulu equipped. 
- Orcs are now recognized as dangerous monsters by NPCs. 
- Hostile NPCs will no longer jitter while the hero is in dialogue. 
- Gate guards can no longer be fooled by having your weapon drawn. 
- NPCs should now properly detect when the player sheathes his weapon. 
- Gate guards will no longer talk to you through walls or at great distances when attacking other NPCs. 
- Friendly NPCs will no longer aggro at the player when he gets attacked (fixes the infamous bug with Gorn in the Free Mine). 
- NPCs will no longer aggro at monsters outside of their attack range, causing them to start running and then immediately stop. 
- NPCs will now detect trespassing, even if they don't see you enter the room. 
- The hero now correctly takes a step back when he is too close while initiating a dialogue. 
- Talking to NPCs who are using an interactive object will no longer require multiple presses of the action button. 
- Fixed broken AI logic so that attacking NPCs search for a weapon if they don't have one in their inventory. 
- NPCs are now able to navigate through doors without getting stuck. 
- The player will no longer get attacked by an NPC for being in the way, if the NPC in question was defeated before. 
- NPCs will no longer get interrupted and then instantly return to their activity when the player is close. 
- NPCs will now correctly steal their opponents ore and weapons. 
- Attacking NPCs now correctly re-equip their melee weapon when the player gets within melee range. 
- Fixed a bug that prevented NPCs from stealing the hero's weapon when defeated. 
- NPCs will no longer initiate dialogue through walls on rare occasions. 
- NPCs will no longer stop attacking when the player leaves the current room under certain conditions. 
- NPCs will no longer point in a random direction when shouting at the hero to come back. 
- Running away from a fight is now the same as losing one. 
- NPCs calling for help will now run away while doing so. 
- Guards will no longer threaten to call the guards when the player is caught trespassing. 
- When fleeing from a fight, NPCs will no longer randomly stop running. 
- Fixed a bug where NPCs wouldn't eat to heal even if they had food in their inventory. 
- When attacking the player after being warned to step aside, NPCs will no longer sheathe and unsheathe their weapon. 
- Friendly NPCs will now verbally respond to being attacked. 
- Fixed an incorrect code sequence that prevented the "Hey you!" and "Just wait!" dialogues from playing. 
- NPCs will no longer get stuck while under the influence of the berzerk spell. 
- The player will no longer get stuck if an enemy casting pyrokinesis on the hero dies mid-cast. 
- Both rice (and soup) eating animations will now play. 
- NPCs no longer need to take a step forward and then backward when interrupted by the player. 
- NPCs will now correctly take a break during cooking. 
- Followers will now properly return to their routine when not in dialogue-mode. 
- Followers should no longer fall off ladders. 
- Guards will now reopen gates if they detect that the player has closed them. 
- Meditating novices now make an "Omm" sound, as they should. 
- NPCs will no longer practice without a weapon. 
- NPCs will now go to the correct waypoint when roasting a scavenger. 
- NPCs are no longer insomniacs and will use beds correctly. 
- NPCs will no longer get stuck in beds when interrupted by the player's footsteps. 
- Smokers now actually smoke. 
- Conversing NPCs will now correctly return to their conversation after being interrupted. 

=== Story fixes ===
- The watermages will now remain hostile in Chapter 6, even after a savegame load. 
- Baal Lukor will now remain hostile, even after a savegame load. 
- Tarrok's and Ur-shak's name changes will now persist across savegames. 
- The Templars at Orc Cemetary are now deceased by default. 
- Fixed several mistyped waypoint names. 
- Fixed incorrect dialogue flags. 
- Kirgo, Herek and Mud will no longer get attacked by other Old Camp members in Chapter 4. 
- The guards in the mages' quarters will no longer attack Bartholo in Chapter 5. 
- Stone will now be hostile in Chapter 5. 
- The key to the storage room at the castle smithy will now appear in Skip's inventory, as it should. 
- The Old Mine's gate now closes properly in Chapter 4. 
- The player can no longer get stuck by touching the winch next to Ashgan without opening the castle gate. 
- The Rice Lord can now be traded with. 
- Fixed several issues with the quest "Water Carrier for Lefty". 
- Horatio will no longer rob you of strength points if you have over 100. 
- Homer's quest can now be finished even without having the lurker claw in your inventory. 
- Gomez will now attack you properly after being spoken to in Chapter 5. 
- Initiating the admission dialogue with Gomez is no longer possible if the player has already joined a guild. 
- Lester will now always go back to his initial position for the start of Chapter 2. 
- You can no longer give the weed delivery to Bartholo more than once. 
- Bartholo can now be traded with. 
- You can no longer ask Bartholo who he is after delivering the weed to him. 
- Bartholo will now attack you properly after being spoken to in Chapter 5. 
- Thorus can no longer be repeatedly bribed. 
- The player can no longer ask Thorus about entering the castle after being allowed to go through the gate. 
- Attempting to get into the castle as a novice without having the necessary swampweed in your inventory will no longer break quest progression. 
- Joining the guards and the fire mages at the same time is no longer possible. 
- Jackal will now accept the player's ore. 
- You can now attempt to offer the ore to Jackal repeatedly if you didn't pay before. 
- Jackal will no longer ask the player for protection money after joining a guild. 
- Corrected incorrect dialogue order. 
- Bullit's dialogue in Chapter 5 will now trigger much more consistently. 
- Scorpio now takes the player's ore when teaching crossbow skills. 
- The BACK and END dialogue options should now always appear last. 
- Scorpio will no longer teach you the crossbow skill if you already have it. 
- Chosing the BACK option when learning crossbow skill will now properly close the menu. 
- Scropio's dialogue about refusing to train the hero is no longer available after chapter 4. 
- The south gate guard can no longer be bribed repeatedly when leading Dusty to the Sect Camp. 
- Removed duplicate "END" dialogues. 
- Bloodwyn will now accept the player's ore. 
- Bloodwyn will no longer ask the player for protection money after joining a guild. 
- Bloodwyn will now confront you at the southern gate of the Old Camp, even in Chapters 5 and 6. 
- The guards at the entrance to the orc territory now guard their post properly. 
- Kirgo now gives beer to the player instead of the other way around. 
- Speaking to Fletcher about Nek after completing the quest "Vanished Guard" will no longer reopen said quest. 
- Fletcher will now confront you at the northern gate of the Old Camp, even in Chapters 5 and 6. 
- You can now attempt to offer a beer to Drake more than once. 
- Y'berion will now attack you if you get into the temple without permission, as he should. 
- Y'berion will no longer reward you if you're a templar. 
- Cor Kalom now makes a comment if the player doesn't have a sufficient level to join the Novices. 
- The quest "The Weed Monopoly" can now be finished even if the bandits are killed. 
- Completing the quest "Taking Weeds to Gomez" will no longer also complete the "The Weed Monopoly" quest. 
- The player can no longer give ore to Cor Kalom more than once, when completing the quest "Taking Weeds to Gomez". 
- The player will no longer receive duplicate log entries when speaking to Cor Kalom as a shadow or a rogue. 
- Certain dialogues with traders and teachers will now create an entry in the information log, as they should. 
- A level requirement is now printed on the screen when trying to join the Templars. 
- Cor Angar will no longer refuse to train you when you're already a templar. 
- Dusty will now behave properly when meeting Baal Tondral. 
- The heavy novice armor is now accessible in Chapter 2. 
- Fixed a softlock that occurred when the player progressed through the "Orc Cemetery" quest too quickly. 
- Damarok will now inform you that he can heal you, after completing the oath of fire. 
- You can no longer ask Corristo to join the mages more than once. 
- Substituted an incorrect dialogue about getting the high robe. 
- You can no longer buy the high robe of fire if you already have it. 
- The player can no longer give the letter to Torrez without having it in his inventory. 
- Torrez will now use standardized dialogue options when teaching mana. 
- You can no longer buy the high robe of water if you already have it. 
- Cronos will no longer give you a message for the firemages if they are dead. 
- Fixed a duplication glitch with Balor and Viran. 
- Nyras will now attack you properly after being spoken to in Chapter 2. 
- The player can now offer ore when Balor asks for it. 
- The hero now responds with Yes/No when asked by Joru about his rations. 
- You can inform Baal Kagan about sharing his weed, only after you have handed out at least one joint. 
- You can now speak to Tarok when you don't have all the parts required for Ulu-Mulu. 
- The Orcish High Priests will now attack you properly after being spoken to. 
- The last orc priest will now talk to you again after charging-up Uriziel. 
- Talking to Lares is now possible, even if Roscoe dies. 
- You can now ask ambient NPCs about Gorn's whereabouts after Lares mentions him in dialogue. 
- The bandit guarding Lares' hut will no longer get the player stuck in a perpetual dialogue, if he's a shadow or a guard. 
- You can no longer get rid of Mordrag by defeating him after he leads you to the New Camp. 
- You can no longer go to the New Camp with Modrag if he's already there. 
- You can now attempt to offer a joint to the New Camp gate guard more than once. 
- Offering a joint to the New Camp gate guard will now unlock extra dialogues. 
- Buster no longer teaches the acrobatics skill if the player has already learned it. 
- The player will now verbally ask Buster what he can teach him. 
- Silas can now be traded with repeatedly. 
- You can no longer kick Shrike from his hut without asking Gorn beforehand. 
- Talking to Sharky about Fisk no longer opens the trade menu. 
- Wolf now teaches dexterity only to New Camp members instead of Old Camp guards. 
- You can no longer bring crawler plates to Wolf after you've already collected the right amount. 
- Cipher can now be talked to even without talking to Balor beforehand. 
- The tavern guard will now let you in when you're a novice, like he says. 
- Offering a joint to the tavern guard will now count for the quest "Canvassing of Customers". 
- The tavern guard will now take your 10 ore if you offer it to him, like he says. 
- Fixed problems with the unused blackmailers. 
- The quest "Shrike's Hut" can no longer be reopened by speaking to Gorn after the fact. 
- Completing the quest "Shrike's Hut" is now possible even if Shrike is killed. 
- Gorn can now be traded with after the convoy raid. 
- The player must now have Cronos' letter in his inventory when delivering it to Milten. 
- Milten will no longer equip Uriziel after the "OREHEAP" Cutscene is finished. 
- The hero will no longer try to equip Uriziel in dialogue, if he doesn't have enough strength. 
- Lester can now guide you around the camp more than once. 
- Diego now makes a comment if the player doesn't have a sufficient level to join the Shadows. 
- The quest "New Fence for Fisk" will trigger even if Mordrag is killed or taken to the New Camp. 
- Swiney will no longer take off his own armor when giving the scraper armor to the player. 
- Cord will no longer offer to teach you level 2 skills, if you haven't learned level 1 skills before. 
- The exit dialogue option will now be always marked as "END" 
- Conflicting dialogue options can no longer be selected when Fisk confronts the hero about Mordrag. 
- Fingers will no longer offer to teach you level 2 skills, if you haven't learned level 1 skills before. 
- Talking to Gor Na Ran after defeating the Minecrawler Queen will no longer create a log entry about joining the templars if you're not a novice. 
- Fixed issues with Gor Na Ran's dialogue in Chapter 6. 
- You can no longer buy the loincloth if you already have it. 
- Gor Boba will now attack you properly after being spoken to in the Sleeper's Temple. 
- Jesse's quest will now trigger properly. 
- Conflicting dialogue options in Jesse's quest can no longer be selected. 
- Grim now correctly states I'm/We're when ambushing the player with a group of diggers. 
- You can now ask Snaf about Nek even after completing the quest "Snaf's Recipe". 
- Aleph now correctly accepts the player's ore. 
- You can no longer buy the key from Aleph more than once. 
- There are no longer more people than there are beds. 
- The guard at the Old Mine entrance now belongs to the mine. 
- Matched Drake's skin color between his head and body. 
- Corrected wrongly assigned NPC IDs to fix a problem with mismatched routines. 
- Cor Kalom should no longer stand around doing nothing on rare occasions. 
- Natalia will no longer keep fanning Y'Berion after his death. 
- Natalia now sits in her own room during the night instead of Chani's. 
- Cronos now has the correct NPC type. 
- Tarrok's health will now be at 10 HP by default, as it should be. 
- A scraper in the Free Mine Camp now has ambient dialogues. 
- The second mercenary guarding the ore mound will now spawn properly. 
- Fixed incorrectly specified waypoints for Cavalorn. 
- The named templars in the Old Mine are now invincible. 

=== Miscellaneous fixes ===
- Fixed incorrectly assigned properties on several items. 
- Items with no value will no longer have a "Value:" string displayed. 
- The Elixir of Healing now uses the correct item model. 
- The book "The Crypt" now has the correct description. 
- Several crystal lights in the Sect Camp now light up at night, as they should. 
- The guard sleeping quarters and surrounding chambers are now off-limits to non-Old Camp members. 
- The player will no longer receive XP for killing previously defeated enemies with magic or a ranged weapon. 
- The unused tramps are now hostile, as they should be. 
- The camera will no longer zoom into the hero's chest when sitting on a bench. 
- Fixed incorrect PFX behavior on the alchemy table's distillation equipment. 
- Babes now have functioning pain and death sounds. 

=== Improvements and additions ===
- Friendly NPCs will now defend themselves when attacked. 
- NPCs will now heal while sleeping. 
- Items on shelves should now be easier to pick up. 
- The unused blackmailers now spawn at the bridge near the New Camp. 
- The unused tramps now spawn. 
- The unused guard guarding the gate of the Old Mine now spawns. 
- Baal Lukor can now be found in the Sect Camp prior to Chapter 3. 
- Gor Boba Lukor can now be found in the Sect Camp prior to Chapter 3. 
- The gate to the exchange place now closes in Chapter 4. 
- The quest "The Big Thing of the Brotherhood" will now resolve in a more elegant manner. 
- You can now ask ambient NPCs about Cor Kalom's whereabouts after Y'Berion mentions him in dialogue. 
- The Sleeper will now wake up from his sleep when Cor Kalom calls upon him. 
- The quest "Ghorim's Replacement" will now fail if Harlok is killed. 
- Adjusted the camera angle so you can now see the Sleeper when impaling his hearts. 


=== Data Fixes ===

=== Meshes ===
- Tree leaves are no longer solid. 
- The falling floor in the old citadel is now aligned properly with the world mesh. 
- The lurker claw will no longer fall through the ground. 
- Fixed a stretched texture on the Orc Axe model. 
- Corrected improper texture on the heavy mercenary armor. 
- The boots on the light mercenary armor now have the correct texture. 

=== Animations ===
- The crossbow's animations are now synced with the player's when shooting. 
- Crossbows are now unloaded when holstered. 
- Jumping with the acrobatic skill will now work consistently. 
- When falling into water with the acrobatic skill, the correct animation will now play. 
- Fixed several scenarios when monsters would freeze when plunging into deep water. 
- NPCs as well as the player will now produce an audible splashing sound when falling into a body of water. 
- The weed stomper sound effect is now synced up with the animation. 
- Fixed the broken roof on Velaya's bed. 
- Replaced broken animation files to make Scavenger rotisseries usable. 
- The fire underneath the cooking cauldron is now textured properly. 
- Fixed incorrect textures on the campfire meshes. 
- The weed stomper object now has the correct textures. 
- Fixed squished shoulders in the lightning victim animation. 
- NPCs heads will no longer squish while eating. 
- The belt on the Light Digger's Trousers is now textured properly. 
- Removed a large dark shadow on The Light Novice's Armor. 
- Fixed stretched texture on the light crossbow model. 
- The bowstring of the Longbow will now retract when shot. 
- The alchemy table is now textured properly. 

=== Textures ===
- Added missing fire animations for the shrines in the Sect camp. 

=== World ===
- Fixed incorrectly mapped textures on several surfaces. 
- Fixed a broken portal in Xardas' tower. As a result, the ground floor now has sharper lighting. 
- Fixed a broken portal in the ore mound cavern. 
- Fixed holes in the New Camp rice field. 
- Fixed holes in the ground which caused the player to fly up into the air. 
- Fixed a hole under the water surface in the swamp. 
- The hero will no longer stumble when walking over the bridge to the Old Camp. 
- Fixed hole in the roof of a cave near Cavalorn's hut. 
- Re-added canyon near the Old Mine. 
- The top of the tower at the old citadel is now accessible. 
- Fixed a broken portal in the Fog Tower dungeon. 
- Adjusted the water surface in the river near the mountain fortress to prevent clipping from the ground underneath. 
- Removed the large portal in the swamp, which was known to cause graphical glitches. 
- Fixed a broken portal in the forest between the Old Camp and the swamp. 
- Repaired and restored the waste outlet on one of the sides of the castle. 
- Increased static lightmap resolution to match Gothic 2. 
- Fixed missing collision on various objects around the world. 
- Fixed invalid sound instances. 
- Locked chests now always have a LOCKED model, and vice versa. 
- Certain gates and movers now make a sound when opening/closing. 
- Fixed missing or incorrect focus texts on interactive objects around the world. 
- The scavenger rotisserie in the Free Mine will now have correct animations when used. 
- Fixed a misplaced campfire in the Old Mine. 
- The lava in the Temple of the Sleeper is now animated. 
- Fixed misplaced items in the Sleeper's Temple. 
- The portal behind the Sleeper is now animated. 
- The target puzzles in the Sleeper's Temple will no longer break on high FPS. 
- The target puzzle with Varrag-Unhilqt can no longer be solved by just walking up to the rear wall. 
- All torch holders now require an active torch to be lit. 
- A chest in the Free Mine Camp will now require a lockpick to open. 
- The grindstone in the New Camp now requires a blade to use. 
- The torch holders near the ore mound now require an active torch to light. 
- The cauldrons in the New Camp now require a scoop to use. 
- Fixed a misplaced table model underneath the map. 
- Fixed misplaced structural logs in the New Camp tavern. 
- The chest in Rice Lord's room now has a valid key. 
- A chest in the new camp can now be lockpicked. 
- Removed faulty objects in the Old Camp. 
- Adjusted several misplaced objects. 
- The door to the storage room at the castle smithy now has a valid key. 
- The water pipes in the house of the ore barons will now have correct animations when used. 
- The flags on the Arena podium will no longer disappear based on the player's position. 
- Added lights to previously unlit dark interiors. 
- A loose board in the Sect Camp now requires a hammer to use. 
- The player can now sleep on all beds in the Sect Camp. 
- Fixed misplaced fire effects in the Sect Camp. 
- Beds in the Sect Camp will now have the correct animations when used. 
- Fixed levitating waterpipe next to Baal Cadar. 
- All campfires are now usable. 
- The gate next to the exchange place is now closable. 
- The gate at the entrance to the New Camp is now closable. 
- The hut at the lake in front of the entrance to the New Camp now has collision. 
- A bunch of winches around the colony now have a properly specified owner guild. 
- Cavalorn will now become angry if you loot his chest. 
- Aidan will now become angry if you loot his chest. 
- The chest in the basement of the stone circle is no longer inside the wall. 
- Removed a misplaced lensflare effect in a cave near the New Camp. 
- The infamous pillar in the old monastery will now fall over in the correct direction. 
- Xardas' tower is now connected to the rest of the world via the waynet. 
- Adjusted misplaced vobs in the Fog Tower dungeon. 
- Fixed incorrenctly placed Fog Tower dungeon lights located at 0,0,0. 
- Adjusted a misplaced column in the Orc City throne room. 
- The collapsing floor of the old citadel will no longer reassemble itself back into place when the button is pressed twice. 
- A tree near the swamp will no longer light up as an interactive object. 
- Fixed misplaced items around the Colony. 
- Fixed an invalid switch target in the Orc City prison. 
</p>
</details>

**Comment by Quintus24** [comment]
This is a fun one "_The unused guard guarding the gate of the Old Mine now spawns._"

It always amazed me how they never bothered to have at least one NPC guarding the gate to the Old Mine.
Now I have my answer. 😃 ❤️ 

Above all, more dialogue lines, vendors and extra spawning creatures. ❤️

**Comment by szapp** [comment]
Do you follow THC on some of their channels? Is this battery of fixes also planned to release for the PC-version?

**Reply by Quintus24** [reply]
Yeah, I follow them on Twitter, the Gothic page at least. (https://twitter.com/gothicthegame)

But I don't know if they plan to release the patch on the PC version.
I did ask them about it in the hopes that one of their moderators would reply, no answer so far.
I guess they are more focused on the Switch port and on the Gothic 1 Remake.

Anyway, If I see more info about it, I will update you guys here!

**Reply by Quintus24** [reply]
Hello,

Returning here with a bit of info, I was reading the comments section about the Switch Trailer and it seems that they might release the patch on PC at some point in the future, no ETA so far.

Here are some screenshots:
![image](https://github.com/AmProsius/gothic-1-community-patch/assets/93062373/7fb025cf-ed55-466f-bd18-2e3da77e8883)

![image](https://github.com/AmProsius/gothic-1-community-patch/assets/93062373/6d240849-aa18-4284-90df-7a4ea0a797cb)

Regards,
Quintus24

**Comment by Quintus24** [comment]
Hello, guys!

The "Patch 1" was released for the Nintendo Switch platform, posting the links here in case they may prove useful.

https://thqnordic.com/article/gothic-classic-patch-1-now-live-nintendo-switch
https://twitter.com/gothicthegame/status/1713872417632739426

No new info regarding the release for the Steam/PC platforms.

Warm Regards,
Quintus24

**Comment by Quintus24** [comment]
Hi, guys!
It's been a while. 😃

Anyway, another Patch for the Switch platform was released, I'm posting the links here in case they may offer some useful info.
Thou from what I see in the notes, there are mostly technical fixes exclusive for the respective platform.

And no, they have still not released these updates on the Steam version, so far. 

https://thqnordic.com/news/g1cnsw103
https://x.com/gothicthegame/status/2059943998668599742

All the best!
Quintus24

**Reply by AmProsius** [reply]
Thank you!

I already had a look at the patch notes earlier today and I couldn't spot an issue related to the game itself (instead of the platform).

We'll keep an eye out though.
